### ISSUE TITLE
[SUGGESTION] Filter Project Template List

### ISSUE THREAD

**Issue by BachiMjavanadze** [microsoft/vscode-dotnettools#261]
Please allow to filter the `Project Template List`

![image](https://github.com/microsoft/vscode-dotnettools/assets/38082501/ff7f9555-0734-488f-98f3-30273b406fe3)

let's say in settings I disabled unused projects.  Next time I should see only enabled projects:

![image](https://github.com/microsoft/vscode-dotnettools/assets/38082501/e7200fa9-86ab-44e8-aee3-e773f59e1902)

**Comment by webreidi** [comment]
You can type in the text box (where it says "Select a template..." to filter down the list manually.

![Image](https://github.com/user-attachments/assets/35db6f1d-5dbc-4ac0-a188-92bdc8f2e959)

@BachiMjavanadze Does this meet your needs or do you want to remove specific projects by some other filters (like not used)?

**Comment by BachiMjavanadze** [comment]
@webreidi
no, my request is completely different; here's what I want: in `settings.json` I should be able to write something like this:
```
  "dotnet.disableProjects": [
    "Console App",
    "ASP.NET Core MVC",
  ]
```

and in the drop-down lists of projects I don't want to see these 2 projects.
